sail_latex_riscv/commands.tex

```diff
@@ -7712,23 +7712,6 @@ \subsection*{Exceptions}
 
 }{\lstinputlisting[language=sail]{sail_latex_riscv/fclCMovezexecute33a689e3a631b9b905b85461d3814943.tex}}}}
 
-\newcommand{\sailRISCVfclClearexecute}{\saildoclabelled{sailRISCVfclClearzexecute}{\saildocfcl{Integer registers 8 $\times$ \emph{q} $+$ \emph{i} are each set to 0 if the \emph{i}th bit
```

Comment:
This should probably be done by re-generating form an updated sail model - but I guess the result will be the same, so should be fine this way too.

Reply:
Please don't touch sail_latex_riscv, just don't use the bits you don't want any more...